controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java

```diff
@@ -132,11 +131,19 @@ protected final void shutDown() throws Exception {
 
                 // First close the reader, which implicitly notifies reader position to the reader group
                 log.info("Closing reader for {}", objectId);
-                reader.close();
+                try {
+                    reader.closeAt(getCheckpoint());
+                } catch (Exception e) {
+                    log.error("Exception while closing EventProcessorCell reader from checkpointStore: ", e);
```

Comment:
Given that we are in the mode of cleaning up unneeded stack traces, I'm wondering why we need to output the stack trace as part of shutting down. 

Also, if this is an "error", then it means that it is unrecoverable, which I can understand because we are not sure about what happened to the reader close call. What's the user supposed to do when seeing this log error messages?